### PR TITLE
SINGA-425 Add 3 operators

### PR DESCRIPTION
**Pull request by Zheng130126**
Add 3 operators , Abs(), Exp() and leakyrelu(), for Autograd Layer class

### CODE REVIEW

**Comment by nudles** [comment]
can you add unit tests like https://github.com/apache/incubator-singa/blob/master/test/python/test_operation.py?

**Comment by Zheng130126** [comment]
I have submitted the unit tests codes.

File: test/python/test_operation.py [hunk 1]
```diff
@@ -264,6 +264,48 @@ def test_MeanSquareError(self):
...
+        self.check_shape(dx.shape(), (3, 2))
+        
+    def test_Exp(self):
+        X=np.array([0.8,-1.2,3.3,-3.6,-0.5,0.5]).reshape(3,2).astype(np.float32)
+        XT=np.array([2.2255409,0.22313017,27.112638,0.02732372,0.60653067,1.6487212]).reshape(3,2).astype(np.float32)
+        x=tensor.from_numpy(X)
+        x.to_device(gpu_dev)
+
+        result=autograd.exp(x)
+        Err=XT-rerult
```
Comment (nudles) [inline test/python/test_operation.py]: rerult --> result?
have you managed to run it?

Comment (Zheng130126) [reply test/python/test_operation.py]: Sorry, The development environment is not suitable, Only run partial code!

**Pull request merged**